typed-racket-lib/typed-racket/typed-racket.rkt

```diff
@@ -16,6 +18,7 @@
          with-type
          (for-syntax do-standard-inits))
 
+(current-inspector new-inspector)
```

Comment:
Is this better:
https://github.com/racket/typed-racket/pull/385/files#diff-bfd28907573d114fba65652f93ae3c05R21
?